src/taco-cli/cli/kit.ts

```diff
@@ -664,27 +666,26 @@ class Kit extends commands.TacoCommandBase {
         if (!semver.valid(version)) {
```

Comment:
Not directly related, but while looking at `npm view` I saw that we could also look at `npm view cordova dist-tags` to get the tags which are not semver valid. Might be worth in future changing the logic here to just invoke `npm view cordova dist-tags versions` and checking to see if the provided value turns up anywhere in there